lib/jekyll/commands/new.rb

```diff
@@ -114,6 +145,11 @@ def site_template
         def scaffold_path
           "_posts/0000-00-00-welcome-to-jekyll.markdown.erb"
         end
+
+        def user_email
```

Comment:
Yes, I added this when I saw that block in theme_builder.rb
But they're both used differently. So DRYing it would be another PR.